modules/repository-s3/src/internalClusterTest/java/org/elasticsearch/repositories/s3/S3RepositoryThirdPartyTests.java

```diff
@@ -48,8 +54,23 @@
 import static org.hamcrest.Matchers.lessThanOrEqualTo;
 import static org.hamcrest.Matchers.not;
 
+@ThreadLeakAction({ ThreadLeakAction.Action.WARN, ThreadLeakAction.Action.INTERRUPT })
+@ThreadLeakScope(ThreadLeakScope.Scope.NONE)
```

Comment:
Why aren't we using `TestContainersThreadFilter` here?

Reply:
fixed